gitensite/apps/content/tests.py

```diff
@@ -25,3 +28,48 @@ def test_view_by_anonymous(self):
         self.assertEqual(r.status_code, 200)
         r = anon_client.get("/license/", follow=True)
         self.assertEqual(r.status_code, 200)
+
+        #Test to ensure that secret is in environment variables
+        self.assertEqual("GITENBERG_SECRET" in os.environ, True)
+
+        yaml = """_repo: testbook
+creator:
+  author:
+    agent_name: Austen, Jane
+    birthdate: 1775
+    deathdate: 1817
+    gutenberg_agent_id: '68'
+    url: http://www.gutenberg.org/2009/agents/68
+    wikipedia: http://en.wikipedia.org/wiki/Jane_Austen
+gutenberg_bookshelf:
+- Best Books Ever Listings
+- Harvard Classics
+gutenberg_issued: '1998-06-01'
+gutenberg_type: Text
+identifiers:
+  gutenberg: '1234223423'
+language: en
+publisher: Project Gutenberg
+rights: Public domain in the USA.
+rights_url: http://creativecommons.org/about/pdm
+subjects:
+- '!lcsh: Love stories'
```

Comment:
this is the wrong syntax.
wrong 
`- '!lcsh: Love stories'`
right
`- !lcsh: 'Love stories'`

Reply:
I see that this is copied from a gitberg test. Let me double check.

Reply:
I fixed this 
an example of correct syntax:

https://raw.githubusercontent.com/gitenberg-dev/metadata/master/samples/pandata.yaml